### PR TITLE
Pkg 2019

### PR DESCRIPTION
**Pull request by markan**
Add at version 1.4.0
        
Required by jupyter server 2.5.0
https://anaconda.atlassian.net/browse/PKG-2019

* Remove unused workflows
* Remove noarch
* Limit to python 3.8 or later
* Linter fixups

Carefully reviewed the https://github.com/kvas-it/pytest-console-scripts/blob/master/setup.py for any changes.

### CODE REVIEW

**Review by ELundby45 — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,26 +1,30 @@
...
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
     - pip
-    - python >=3.6
-    - setuptools-scm
+    - python
+    - setuptools
+    - setuptools-scm 7.1.0
```
Comment (Jrice1317) [inline recipe/meta.yaml:22]: To clarify my understanding and reiterate the discussion in our meeting this morning, `conda-build` will automatically pull the latest version we have of `setuptools-scm`, which in this case is `7.1.0`, so we are pinning it to that version for transparency?

Comment (markan) [reply recipe/meta.yaml:22]: I might have missed that discussion. The linter complains about it not being pinned. Is that a false positive?

Comment (marcoesters) [reply recipe/meta.yaml:22]: It probably should be an exception, but pinning it won't hurt.

**Review by skupr-anaconda — APPROVED** [review]
LGTM

**Pull request merged**